test/SlowTests/Core/Indexing/ReferencedDocuments.cs

```diff
@@ -27,65 +26,6 @@ namespace SlowTests.Core.Indexing
     public class ReferencedDocuments : RavenTestBase
     {
         [Fact]
-        public void CanUseLoadDocumentToIndexReferencedDocs()
```

Comment:
Restore candidate: here we test more indexing of referenced docs that transformers.